Source/NujasAI/Public/NujasAIGlobals.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2018-2019 The Janus Project | 2034 Complex LLC. All Rights Reserved.
+
+#pragma once
+
+#include "Interface.h"
+#include "NujasAIGlobals.generated.h"
+
+UINTERFACE(Blueprintable)
+class NUJASAI_API UAggressive : public UInterface
```

Comment:
There are a total of 96 results for ` : public UInterface`  https://github.com/EpicGames/UnrealEngine/search?q=%22+%3A+public+UInterface%22&unscoped_q=%22+%3A+public+UInterface%22

There are 59 resutls for `Interface : public UInterface`
https://github.com/EpicGames/UnrealEngine/search?q=%22Interface+%3A+public+UInterface%22&unscoped_q=%22Interface+%3A+public+UInterface%22

This shows that the naming convention with the variable name postfixed with `Interface` is the majority

Since the PlatformInterface we have is also using the majority convention: https://github.com/nujas/TheJanusFramework/blob/f7f455342b102c86f840e0c201f9d1dbb2295e33/Source/NujasCore/Public/PlatformInfoInterface.h

I would suggest rename this to `UAggressiveInterface`